clients/logging/formatter/json.py

```diff
@@ -1,7 +1,7 @@
 import datetime
 import logging
 
-import helpers
+import clients.logging.formatter.helpers as helpers
```

Comment:
```suggestion
import clients.logging.formatter.helpers
```

And change line 38 to us for namespace for helpers please
We try to keep `as alias` to only where it's needed. Here the internal formatter helpers are just used once... not really saving anything hiding the namespace